src/drivers/github.js

```diff
@@ -248,8 +248,14 @@ class Github {
   async runnerByName(opts = {}) {
```

Comment:
Dryer like below.

Reply:
What is _below?_ 👀

Reply:
above!! For some reason I thought that I was commenting the chunk on top